src/Core/Framework/Adapter/Twig/Filter/CurrencyFilter.php

```diff
@@ -31,7 +32,14 @@ public function getFilters()
     }
 
     /**
+     * @param array<string, mixed> $twigContext
```

Comment:
also please use native types or "deprecate" them, so we have them in the next major